api/db/populate_users.js

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/env node
+
+// TODO: This will all be in the general js to abide by DRY principal
+const userArgs = process.argv.slice(2);
+
+if (!userArgs[0].startsWith('mongodb')) {
+  console.log(
+    'Error: you need to specify a valid MongoDB URL as the first argument'
+  );
+
+  return;
+}
+
+const User = require('../models/User');
+const Interest = require('../models/Interest')
+const Token = require('../models/Token')
+const crypto = require('crypto');
+
+const async = require('async');
+
+const mongoose = require('mongoose');
+const mongoDB = userArgs[0];
+mongoose.connect(mongoDB, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = global.Promise;
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'MongoDB connection error:'));
+
+const { uniqueNamesGenerator, names } = require('unique-names-generator');
+const countries = require('country-region-data');
+const locales = ['en', 'es'];
+const users = [];
+
+const createUser = async (
+  name, email, emailConfirmed, emailConfirmedAt,
+    dateOfBirth, country, region,
+    preferredLocale, interests, admin,
+    callback
+) => {
+  const user = new User({
+    name, email, emailConfirmed, emailConfirmedAt,
+    dateOfBirth, country, region,
+    preferredLocale, interests, admin
+  });
+  user.setPassword('Password1');
```

Comment:
```suggestion
  user.setPassword('password');
```
I think it would be better to use `'password'` here, as it's easier to remember.

Reply:
I can do that. Are you planning to make password requirements anytime in the future (ie. must contain 1 letter, 1 number, 1 special character)? If so, I would recommend we set the password to be in line with those potential requirements now rather than having a potential issue in the future.

Reply:
I have no plans to do that as of now, so let's keep it simple for the time being.